### ISSUE TITLE
Diceloss cannot effect  under softmax with keras

### ISSUE THREAD

**Issue by AbnerxzHe** [ellisdg/3DUnetCNN#208]
Hi, sir. when I used softmax instead of sigmoid. and my final output is (1,128,128,128,4) ,the 4 is the channel. When I use dice to train the model as an objective function with keras，I find that the model does not learn to converge. I try many ways,but the problem is still.If you can, please give me some advice.
